modules/self-test/test_helpers.lua

```diff
@@ -44,7 +44,10 @@
 		return prj
 	end
 
-
+	function m.createGroup(wks)		
+		local prj = group ("MyGroup" .. (#wks.groups + 1))
```

Comment:
just nitpicking I guess... with typically we use 'prj' for projects...
it's a local, so not a big deal, but I would probably rename that local to grp or something.